ui/src/components/VolumeOverviewTab.js

```diff
@@ -195,6 +210,8 @@ const VolumeDetailCard = (props) => {
     <VolumeDetailCardContainer>
       <VolumeInformation>
         <VolumeNameTitle data-cy="volume_detail_card_name">
+          <CircleStatus className="fas fa-circle fa-2x" status={health} />
```

Comment:
Ha. If you use the component `<CircleStatus/>` the double size may not work.

You may want to cherry-pick [this commit](https://github.com/scality/metalk8s/pull/2868/commits/778b284d107e87614d1568af1ebd3b6225793f1a) (once it's merged) and properly use this component.

Reply:
I tried with your commit but it seems the CIRCLE_SIZE_... constants are not included in the commit. I removed the 2x for now it seems it is still displaying properly :) . Will update when your commit will be merged.